test/cpp/end2end/orca_service_end2end_test.cc

```diff
@@ -107,16 +112,13 @@ class OrcaServiceEnd2endTest : public ::testing::Test {
   ~OrcaServiceEnd2endTest() override { server_->Shutdown(); }
 
   std::string server_address_;
-  OrcaService orca_service_;
+  ServerMetricRecorder server_metric_recorder_;
+  std::unique_ptr<OrcaService> orca_service_;
   std::unique_ptr<Server> server_;
   std::unique_ptr<OpenRcaService::Stub> stub_;
 };
 
 TEST_F(OrcaServiceEnd2endTest, Basic) {
-  constexpr char kMetricName1[] = "foo";
```

Comment:
Please re-add the code for testing named metrics.

Reply:
Done.